caseworker-api/composer.json

```diff
@@ -48,7 +48,7 @@
         "laminas/laminas-permissions-rbac": "^2.5",
         "laminas/laminas-servicemanager": "^3.3",
         "laminas/laminas-stdlib": "^3.1",
-        "laminas/laminas-dependency-plugin": "^1.0"
+        "laminas/laminas-dependency-plugin": "^2.0"
```

Comment:
Does this plugin determine the composer version used? I'm not sure how this relates to a composer upgrade atm.

Reply:
the composer container image is latest, which is at 2.0. this dependancy plugin needed to be upgraded to support that. we don't pin versions unless needed generally on the container images, unless its something like php for example.

Reply:
:+1: fine, think we can count this as resolved then